src/main.js

```diff
@@ -217,168 +226,90 @@ const renderUi = () => {
-          startY + 70,
-          startX + 250,
-          startY + 40,
-          startX + 220,
-          startY + 20
-        );
-        cx.bezierCurveTo(
-          startX + 260,
-          startY - 40,
-          startX + 200,
-          startY - 50,
-          startX + 170,
-          startY - 40
-        );
-        cx.bezierCurveTo(
-          startX + 150,
-          startY - 75,
-          startX + 80,
-          startY - 60,
-          startX + 80,
-          startY - 40
-        );
-        cx.bezierCurveTo(
-          startX + 30,
-          startY - 75,
-          startX - 20,
-          startY - 60,
-          startX,
-          startY
-        );
-        cx.closePath();
-        cx.fill();
-        cx.restore();
+const createCloud = (y, z, opacity) => {
+  return Sprite({
+    x: Math.random() * level.width * (5 / 4) - level.width / 4,
+    y:
+      z === 0
+        ? y + (Math.random() * 200 - 102120)
+        : y + (Math.random() * 200 - 100),
+    color: "white",
+    opacity: z === 0 ? 0.95 * opacity : 0.7 * opacity,
+    dx: z === 0 ? 0.07 + Math.random() * 0.1 : 0.05 + Math.random() * 0.1,
+    radius: 20 + Math.random() * Math.random() * 70,
+
+    update: function() {
+      this.advance();
+      if (this.x - 300 > level.width) {
+        this.x = -600;
       }
-    });
-  } else {
-    return Sprite({
-      x: random(level.width * (5 / 4)) - level.width / 4,
-      y: y + (random(200) - 100),
-      color: "white",
-      opacity: 0.7,
-      dx: 0.05 + random(0.1),
-      radius: 20 + random() * random() * 70,
-
-      update: function() {
-        this.advance();
-        if (this.x - 300 > level.width) {
-          this.x = -600;
-        }
-      },
+    },
 
-      render: function() {
-        let cx = this.context;
-        cx.save();
-        cx.fillStyle = this.color;
-        (cx.globalAlpha = this.opacity), cx.beginPath();
-        cx.beginPath();
-        let startX = this.x;
-        let startY = this.y;
-        cx.moveTo(startX, startY);
-        cx.bezierCurveTo(
-          startX - 40,
-          startY + 20,
-          startX - 40,
-          startY + 70,
-          startX + 60,
-          startY + 70
-        );
-        if (this.radius < 45) {
-          cx.bezierCurveTo(
-            startX + 80,
-            startY + 100,
-            startX + 150,
-            startY + 100,
-            startX + 170,
-            startY + 70
-          );
-        }
-        cx.bezierCurveTo(
-          startX + 250,
-          startY + 70,
-          startX + 250,
-          startY + 40,
-          startX + 220,
-          startY + 20
-        );
-        cx.bezierCurveTo(
-          startX + 260,
-          startY - 40,
-          startX + 200,
-          startY - 50,
-          startX + 170,
-          startY - 40
-        );
+    render: function() {
+      let cx = this.context;
+      cx.save();
+      cx.fillStyle = this.color;
+      (cx.globalAlpha = this.opacity), cx.beginPath();
```

Comment:
An extra `beginPath` here?

Reply:
Some typo plus eslint oddness, will be fixed.